tasks/db_deploy/test/manual_tests/README.md

```diff
@@ -772,7 +661,8 @@ PostgreSQL *disaster_recovery* database. Perform the checks below by going to th
 
 No cleanup is necessary if the next test run is the [Database No Migration Test](#database-no-migration-test).
 
-The `sql/cleanup.sql` script will remove all objects including the *disaster_recovery* database. The
+To cleanup from this test you can use one of two scripts. The `sql/cleanup.sql`
```

Comment:
Possibly revert back to Rizbi words here in paragraph.

Reply:
Reverted merge conflict